src/pages/groupComparison/ProteinEnrichments.tsx

```diff
@@ -17,34 +18,44 @@ export interface IProteinEnrichmentsProps {
 @observer
 export default class ProteinEnrichments extends React.Component<IProteinEnrichmentsProps, {}> {
     @autobind
-    private onChangeProfile(m:MolecularProfile) {
-        this.props.store.setProteinEnrichmentProfile(m);
+    private onChangeProfile(profileSet:{[id:string]:MolecularProfile}) {
```

Comment:
profileMap?  I think it's good for us to get specific about the terminology since there are now these primitives in the language